opentelemetry-zio-logging/src/main/scala/zio/telemetry/opentelemetry/zio/logging/TelemetryLogFormats.scala

```diff
@@ -0,0 +1,39 @@
+package zio.telemetry.opentelemetry.zio.logging
+
+import io.opentelemetry.api.trace.{Span, SpanContext}
+import io.opentelemetry.context.Context
+import zio.FiberRefs
+import zio.logging.LogFormat
+import zio.telemetry.opentelemetry.context.ContextStorage
+
+object TelemetryLogFormats {
+
+  /**
+   * Will print traceId from current span or nothing when not in span
+   */
+  def traceId(contextStorage: ContextStorage): LogFormat = LogFormat.make { (builder, _, _, _, _, _, fiberRefs, _, _) =>
+    getSpanContext(contextStorage, fiberRefs).map(_.getTraceId) match {
+      case Some(traceId) => builder.appendText(traceId)
+      case None          => ()
+    }
```

Comment:
nitpick: use `.fold` to make code more concise:
```
getSpanContext(contextStorage, fiberRefs).map(_.getTraceId).fold(())(builder.appendText)
```

Reply:
Fixed